apps/resources/views.py

```diff
@@ -172,3 +175,34 @@ def get_context_data(self, *args, **kwargs):
         )
         context['featured_resources'] = featured_resources
         return context
+
+
+@require_POST
+@staff_member_required
+@permission_required('resources.change_resource')
+def admin_categorise_resources_view(request):
+    """
+    Admin Action for assigning categories to resources.
+    """
+    resources = Resource.objects.filter(id__in=request.POST.get('resource_ids', '').split(','))
+    categories = ResourceCategory.objects.filter(id__in=request.POST.getlist('category_ids', []))
+    remove_categories = request.POST.get('remove') == '1'
```

Comment:
would it be worth using `from distutils.util import strtobool`?

Reply:
I don't think it really adds anything... the checkbox value is explicitly set to 1 if checked...